src/chi/components/stat/stat.scss

```diff
@@ -111,6 +111,10 @@
           left: $stat-background-icon-icon-left;
           position: relative;
           width: 4.5rem;
+
+          &.outline {
```

Comment:
Since this is a complimentary class, do we need a `-` prefix here?
@mattnickles @jllr

Reply:
@janibolkvadze This is targeting an existing class in the marketing icon web component which does not have a `-` prefix. Agree it makes sense to rename but I would recommend doing that in a separate PR.

Reply:
Sounds good